applications/acdc/test/test.config

```diff
@@ -0,0 +1,10 @@
+[
+ {lager, [
+          {handlers, [
+                      {lager_console_backend, '=info'}
```

Comment:
On that branch's Travis CI runs you can see that lager prints no logs... but that may be because lager is not started / tests don't get compiled with the lager transform. I'll investigate later :)